spec/workers/content_changes_worker_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe ContentChangesWorker do
+  describe ".perform" do
```

Comment:
I wonder if we should add a test on `perform_async` to check it gets added to the right queues similar to https://github.com/alphagov/email-alert-api/blob/63799a71b45c869ffa009543ff9f8ba38ce1554b/spec/workers/subscription_contents_worker_spec.rb#L89-L99.

Reply:
Thanks, I've added this test